packages/docusaurus-plugin-content-blog/src/__tests__/index.test.ts

```diff
@@ -526,3 +526,64 @@ describe('blog plugin', () => {
     expect(blogPosts).toMatchSnapshot();
   });
 });
+
+describe('last update', () => {
+  const siteDir = path.join(
+    __dirname,
+    '__fixtures__',
+    'website-blog-with-last-update',
+  );
+  it('author and time', async () => {
+    const plugin = await getPlugin(
+      siteDir,
+      {
+        showLastUpdateAuthor: true,
+        showLastUpdateTime: true,
+      },
+      DefaultI18N,
+    );
+    const {blogPosts} = (await plugin.loadContent!())!;
+
+    expect(blogPosts).toMatchSnapshot();
```

Comment:
Test cases look good, but using snapshots makes it harder to review because the result is not colocated with the test.

Since we are testing a specific feature here, I'd prefer to use assertions such as `blogPosts[0].metadata.lastUpdateAt === new Date("01/01/2020).getTime()`

But we can keep it this way 👍, snapshots capture that the feature is "wired correctly, and we are going to add tests with assertions for the `readLastUpdateData` code directly in docusaurus utils. 

docusaurus-utils will be a better place to test things in depth because it would be pure unit tests (no IO/fixture/site/plugin/loadContent), and this code is shared between docs/blog so if we tested things in depth in each plugin we end up with duplicated tests somehow.